examples/AccountManagement/ApproveMerchantCenterLink.php

```diff
@@ -0,0 +1,197 @@
+        }
+    }
+
+    /**
+     * Runs the example.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the customer ID
+     * @param int $merchantCenterAccountId the Merchant Center account ID
+     */
+    public static function runExample(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        int $merchantCenterAccountId
+    ) {
+        // Lists all merchant links of the specified customer ID.
+        $merchantCenterLinkServiceClient = $googleAdsClient->getMerchantCenterLinkServiceClient();
+        $response = $merchantCenterLinkServiceClient->listMerchantCenterLinks($customerId);
+        printf(
+            "%d Merchant Center link(s) found with the following details:%s",
+            $response->getMerchantCenterLinks()->count(),
+            PHP_EOL
+        );
+        foreach ($response->getMerchantCenterLinks() as $merchantCenterLink) {
+            /** @var MerchantCenterLink $merchantCenterLink */
+            printf(
+                "Link '%s' has status '%s'.%s",
+                $merchantCenterLink->getResourceName(),
+                MerchantCenterLinkStatus::name($merchantCenterLink->getStatus()),
+                PHP_EOL
+            );
+            // Approves a pending link request for a Google Ads account with the specified customer
+            // ID from a Merchant Center account with the specified merchant center account ID.
+            if (
+                $merchantCenterLink->getIdUnwrapped() === $merchantCenterAccountId
+                && $merchantCenterLink->getStatus() === MerchantCenterLinkStatus::PENDING
+            ) {
+                // Updates the status of Merchant Center link to 'ENABLED' to approve the link.
+                self::updateMerchantCenterLinkStatus(
+                    $merchantCenterLinkServiceClient,
+                    $customerId,
+                    $merchantCenterLink,
+                    MerchantCenterLinkStatus::ENABLED
+                );
+                // There is only one MerchantCenterLink object for a given Google Ads account and
+                // Merchant Center account, so we can break early.
+                break;
+            }
+        }
+    }
+
+    /**
+     * Updates the status of a Merchant Center link with a specified merchant center link status.
+     *
+     * @param MerchantCenterLinkServiceClient $merchantCenterLinkServiceClient the Merchant Center
+     *     link service client
+     * @param int $customerId the customer ID
+     * @param MerchantCenterLink $merchantCenterLink the Merchant Center link to update
+     * @param int $merchantCenterLinkStatusToUpdate the  status to be updated to
+     */
+    private static function updateMerchantCenterLinkStatus(
+        MerchantCenterLinkServiceClient $merchantCenterLinkServiceClient,
+        int $customerId,
+        MerchantCenterLink $merchantCenterLink,
+        int $merchantCenterLinkStatusToUpdate
```

Comment:
Can we simply name it as "status"?

https://github.com/googleads/google-ads-java/blob/master/google-ads-examples/src/main/java/com/google/ads/googleads/examples/shoppingads/ApproveMerchantCenterLink.java#L156

Reply:
I prefer detailed variable names.